libraries/AP_HAL_PX4/RCOutput.cpp

```diff
@@ -329,7 +331,12 @@ void PX4RCOutput::write(uint8_t ch, uint16_t period_us)
         _max_channel = ch + 1;
     }
 
+
```

Comment:
There's one unnecessary line added.. probably a mistake so I'd remove it although obviously not a big deal.